README.md

```diff
@@ -208,7 +208,10 @@ The list of additional options available for running Standardization:
 | --csv-quote **character**  | Specifies a character to be used as a quote for creating fields that might contain delimiter character. By default `"` is used. |
 | --csv-escape **character** | Specifies a character to be used for escaping other characters. By default '&#92;' (backslash) is used. |
 | --trimValues **true/false** | Indicates if string fields of fixed with text data should be trimmed.        |
+| --is-xcom **true/false**   | If `true` a mainframe input file is expected to have XCOM RDW headers.        |
 | --folder-prefix **prefix** | Adds a folder prefix before the date tokens.                                  |
+| --experimental-mapping-rule **true/false** | If `true`, an experimental optimized mapping rule implementation is used. The default value is environment-specific and is set in configuration files. |
```

Comment:
Which configuration files are you talking about?

Reply:
application.properties in Conformance:
```
# Use experimental mapping rule implementation that groups explosions for rules
# operating on the same array
conformance.mapping.rule.experimental.implementation=false

# Enable workaround for Catalyst execution plan optimization freeze
conformance.catalyst.workaround=true
```

Reply:
Then it is a build specific, not environment specific. I view these 2 as quite different. One is easy to change, the second one needs a full new build. For this, we have #510 for extraction of properties for Menas. Maybe create something even for standardization and conformance?

Reply:
Ok, will change to 'build specific'.